### PR TITLE
feat/US34316: [UI] Governance DAO Recent Proposals 

### PR DESCRIPTION
**Pull request by anshulkhare-lab49**
**Description**:
Added Recent proposals in Governance DAO Dashboard
- Created Recent Proposal component and used it for MultiSig and Governance DAO
Screenshot 2023-07-04 at 12 32 18 PM

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboardOverview.tsx [hunk 1]
```diff
@@ -127,9 +132,14 @@ export function GovernanceDAODashboardOverview() {
           </GridItem>
         </Grid>
       </Flex>
-      <Flex gap="4" direction="column">
-        <Text textStyle="h4 medium">Recent Transactions</Text>
-      </Flex>
+      <RecentProposals
+        proposals={proposals?.sort((proposalA, proposalB) => +proposalB.timestamp - +proposalA.timestamp).slice(0, 3)}
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboardOverview.tsx:136]: I recommend moving display calculation/business logic above the return function within React components: 

`const recentProposals = proposals?.sort((proposalA, proposalB) => +proposalB.timestamp - +proposalA.timestamp).slice(0, 3)`

This helps with component maintenance and debugging as a component grows in complexity.

File: src/dex-ui/pages/dao/MultiSigDAODashboard/DashboardOverview.tsx [hunk 2]
```diff
@@ -113,12 +74,14 @@ export function DashboardOverview() {
...
         </Grid>
       </Flex>
-      <Flex gap="4" direction="column">
-        <Text textStyle="h4 medium">Recent Proposals</Text>
-        <Flex direction="column" gap="2" minHeight="300px">
-          {RecentProposalsList}
-        </Flex>
-      </Flex>
+      <RecentProposals
+        proposals={proposals?.sort((proposalA, proposalB) => proposalA.id - proposalB.id).slice(0, 3)}
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/MultiSigDAODashboard/DashboardOverview.tsx:78]: See comment above.

File: src/dex-ui/pages/dao/RecentProposals.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        <Flex direction="column" gap="2" minHeight="300px">
+          {proposals.map((transaction, index) => (
+            <ProposalCard proposal={transaction} dao={dao} key={index} />
+          ))}
+        </Flex>
+      </Flex>
+    );
+  }
+
+  return (
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/RecentProposals.tsx:50]: The "Recent Proposal" header should be present when the Not Found message is displayed.

Current UI (missing Recent Proposal section header):
Screen Shot 2023-07-05 at 1 36 39 PM

Expected:
Screen Shot 2023-07-05 at 1 39 54 PM

Comment (damiano-melcarne-lab49) [reply src/dex-ui/pages/dao/RecentProposals.tsx:50]: The UI design and experience for this section should match across all DAO dashboards.

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]

File: src/dex-ui/pages/dao/MultiSigDAODashboard/DashboardOverview.tsx (hunk 2 above)
Comment (anshulkhare-lab49) [reply src/dex-ui/pages/dao/MultiSigDAODashboard/DashboardOverview.tsx:78]: made the suggested changes.

File: src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboardOverview.tsx (hunk 1 above)
Comment (anshulkhare-lab49) [reply src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboardOverview.tsx:136]: Thanks for the feedback. made the suggested changes.

File: src/dex-ui/pages/dao/RecentProposals.tsx (hunk 3 above)
Comment (anshulkhare-lab49) [reply src/dex-ui/pages/dao/RecentProposals.tsx:50]: 1. Added the missing heading in the Recent proposals. 
2. Using same component for both the places.

Comment (anshulkhare-lab49) [reply src/dex-ui/pages/dao/RecentProposals.tsx:46]: Screenshot 2023-07-06 at 4 40 50 PM
Screenshot 2023-07-06 at 4 41 39 PM

**Review by roshan-bisht-lab49 — APPROVED** [review]

**Review by damiano-melcarne-lab49 — APPROVED** [review]

**Review by roshan-bisht-lab49 — APPROVED** [review]

**Pull request merged**